data/cache/.gitignore

```diff
@@ -1,2 +0,0 @@
-*
```

Comment:
This file seems to have been deleted by accident. Do you mind putting it back?

Reply:
Yes, will do.